code/game/mecha/combat/durand.dm

```diff
@@ -14,6 +14,19 @@
 	var/defence = 0
 	var/defence_deflect = 35
 	wreckage = /obj/effect/decal/mecha_wreckage/durand
+	mech_sprites = list(
+		"durand",
+		"old_durand",
+		"gator",
+		"dollhouse"
+	)
+	paintable = 1
+	var/list/sprites_with_a_lockdown_state= list(
```

Comment:
instead of fussing with these lists, why not just ensure there is SOME kind of lockdown and speedmode state, etc. for each mecha?

even if it's really simple or something, then we can eschew these lists and have consistency.

Reply:
yeah sure, but what about dmi limits to icon_states. This would delay the inevitable icon bloat I guess

Reply:
I seriously doubt mecha is getting anywhere near the limit, it's really high

Even then I don't think that's a reason to just not have an icon (for example, it would seem absurd to have a mech that just doesn't have a broken state, right?)